meerk40t/core/elements.py

```diff
@@ -2964,4 +3223,10 @@ def my_sign(x):
                 pt_older = pt_old
                 pt_old = pt
             after = len(obj.points)
+            time_pass1 = time()
```

Comment:
left over debug code. If we aren't printing we don't need this either.

Reply:
Removed